### ISSUE TITLE
Error when setting Standheizung to true

### ISSUE THREAD

**Issue by el-Presi** [TA2k/ioBroker.vw-connect#164]
**Describe the bug**  
When setting "vw-connect.0.xyz.remote.standheizung" to "false" (via admin, and also via script), i get the error message:

```
- error 403
- error {"error":{"errorCode":"RS.security.9007","description":"The context has not gained a legitimation due to missing access rights."}}
- error failed set state
```

--> I will get no error, when changing it to "true", but nevertheless, the heater is still not turned on. 1. The car doesn't do anything, 2. the "vw-connect.0.xxyz.remoteStandheizung.climatisationStateReport.climatisationState" does not change and the remaining neither does.

**Versions:**  
 - Adapter version: v0.0.45
 - JS-Controller version: 3.3.18
 - Node version: v12.22.7
 - Operating system: Ubuntu 21.04

**Comment by TA2k** [comment]
Did you try all standheizung and climatisaion states?

**Comment by el-Presi** [comment]
yes - it only worked (and only sometimes) with the normal one (non v2)...

BUT: it seems to work way much better with a fresh, new ioBroker installation - I've done that now, and keep on tracking, if it works :)

**Comment by el-Presi** [comment]
again new and fresh ioBroker installation from today... I am now not able to do anything. I cannot activate the heater (Standheizung)... not via v1, not via v2 not vie ventilation start - not at all. It worked (sometimes) before already..!
>> Yes, enough fuel is in - battery charged well - activation via key or SkodaApp works...
Whats wrong with my car? Or is it an issue with this adapter?

**Comment by TA2k** [comment]
I think it is wrong with the adapter but I need an account to see what the app is doing different

**Comment by THEIDEACOMPANY** [comment]
> I think it is wrong with the adapter but I need an account to see what the app is doing different

Did you got an account?

**Comment by TA2k** [comment]
Not yet

**Comment by THEIDEACOMPANY** [comment]
Do you want to use my account?
Not sure if you're a German guy.... Otherwise I would write in german...
Andreas

Am Di., 11. Jan. 2022 um 21:55 Uhr schrieb TA2k ***@***.***>:

**Comment by TA2k** [comment]
schreib mir eine mail tombox2020@gmail.com

**Comment by THEIDEACOMPANY** [comment]
Gemacht....

**Comment by 0x1234567890** [comment]
Habe nun heute auch das Problem. Warum brauche ich einen anderen Account dafür?
